### PR TITLE
Added Central Kurdish (Sorani) language.

### PR DESCRIPTION
**Pull request by swarakaka**

### CODE REVIEW

**Comment by felixxm** [comment]
@swarakaka Thanks :+1: The `ckb` language is already partly translated on [Transifex](https://docs.djangoproject.com/en/dev/internals/contributing/localizing/#translations). Can you fill missing translations via Transifex first?

**Comment by swarakaka** [comment]
@felixxm Thank you very much for your notice. I have requested to join the project on the transifex website.

**Comment by swarakaka** [comment]
> 

I would appreciate you, if you could approve my request. So that I can get the translations done as soon as possible.

**Comment by felixxm** [comment]
> @felixxm
> 
> I would appreciate you, if you could approve my request. So that I can get the translations done as soon as possible.

CC-ing @kosartofiq as a team manager of the Central Kurdish language.

**Comment by swarakaka** [comment]
Hello @felixxm 
all strings have now been translated into Sorani Kurdish, except those that cannot be translated.

**Comment by swarakaka** [comment]
Hello @felixxm
As I wrote earlier, the translation project is complete. What else do I need to do? Do you have a specific time frame for merging translated languages?

**Comment by felixxm** [comment]
@swarakaka Thanks :+1: We normally add languages before the final release, so this is targeted to Django 4.2.

Please remove the `django/conf/locale/ckb/LC_MESSAGES` directory (as all translations are synchronize separately before the release), add `ckb` (Central Kurdish) to the `LANGUAGES` in `django/conf/global_settings.py` and `django/conf/locale/__init__.py`, and English translations. You can check a similar change for the Malay language, 5e218cc0b704ebf64c460050a97b5fafe63e92b0.

**Comment by swarakaka** [comment]
@felixxm Thank you, too.
I made the changes as per your requirements.

File: django/conf/locale/__init__.py [hunk 1]
```diff
@@ -73,6 +73,12 @@
         "code": "ca",
         "name": "Catalan",
         "name_local": "català",
+    },
+     "ckb": {
+        "bidi": False,
```
Comment (felixxm) [inline django/conf/locale/__init__.py:78]: Is Sorani not a RTL language?
```suggestion
        "bidi": True,
```
We should also add it to the `LANGUAGES_BIDI` in the `global_settings.py`

File: django/conf/global_settings.py [hunk 2]
```diff
@@ -65,6 +65,7 @@ def gettext_noop(s):
     ("br", gettext_noop("Breton")),
     ("bs", gettext_noop("Bosnian")),
     ("ca", gettext_noop("Catalan")),
+    ("ckb", gettext_noop("Kurdish Sorani")),
```
Comment (felixxm) [inline django/conf/global_settings.py:68]: Maybe
```suggestion
    ("ckb", gettext_noop("Central Kurdish (Sorani)")),
```

File: django/conf/locale/__init__.py (hunk 1 above)
Comment (swarakaka) [reply django/conf/locale/__init__.py:78]: Yes, Kurdish Sorani is an RTL language. Thanks.

File: django/conf/locale/__init__.py [hunk 3]
```diff
@@ -73,6 +73,12 @@
         "code": "ca",
         "name": "Catalan",
         "name_local": "català",
+    },
+     "ckb": {
```
Comment (felixxm) [inline django/conf/locale/__init__.py:77]:
```suggestion
    "ckb": {
```

**Comment by felixxm** [comment]
@swarakaka Thanks :+1: I pushed small edits.

@claudep Translations will be synced before the final 4.2 release but `ckb` is now 100% translated, so I think we can merge it even now. Does it work for you?

**Comment by claudep** [comment]
Sure, we can do that!

**Comment by felixxm** [comment]
@swarakaka Welcome aboard :boat: 

Merged in 08303f4f06ae25c1bb4d857de1beacceb9dd18fd.

**Comment by felixxm** [comment]
@swarakaka There is an issue with the abbreviated form of the days of the week in Sorani language. They are the same as non-abbreviated forms and don't fit the calendar widget:

image

Can you fix them?

https://github.com/django/django/blob/1d1ddffc27cd55c011298cd09bfa4de3fa73cf7a/django/conf/locale/ckb/LC_MESSAGES/django.po#L897-L937

**Comment by swarakaka** [comment]
Hi @felixxm, 
thank you for your warning. I corrected those on the Transifex.

**Comment by felixxm** [comment]
> Hi @felixxm, thank you for your warning. I corrected those on the Transifex.

Thanks :+1: Can you fix also one-letter forms in:

https://github.com/django/django/blob/073b5fd40035759afaebe5c77de5269aee851643/django/contrib/admin/locale/ckb/LC_MESSAGES/djangojs.po#L247-L273

**Comment by swarakaka** [comment]
I adjusted them too.

**Comment by felixxm** [comment]
> I adjusted them too.

Thanks, see #16719.

**Pull request closed without merging**